build-system/pr-check.js

```diff
@@ -24,6 +24,7 @@
  * This script attempts to introduce some granularity for our
  * presubmit checking, via the determineBuildTargets method.
  */
+const atob = require('atob');
```

Comment:
TIL that [Node doesn't provide this](https://github.com/nodejs/node/issues/3462).